lib/blueberry_rails/app_builder.rb

```diff
@@ -4,12 +4,15 @@ class AppBuilder < Rails::AppBuilder
     include BlueberryRails::ActionHelpers
 
     def readme
-      template 'README.md.erb', 'README.md'
+      copy_file 'README.md', 'README.md'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.